scripts/edge_restore_bacnet_inventory.sh

```
@@ -0,0 +1,53 @@
+fi
+[[ -n "$HOST" ]] || { echo "Set EDGE_HOST or pass --host" >&2; exit 1; }
+
+REMOTE="${ANSIBLE_REMOTE_USER:-ben}@${HOST}"
+REMOTE_DIR="${OPENFDD_REMOTE_DIR:-/home/ben/open-fdd}"
+
+echo "Copying BACnet inventory to ${REMOTE}..."
+scp "$DISC" "${REMOTE}:${REMOTE_DIR}/workspace/bacnet/commissioning/points_discovered.csv"
+[[ -f "$PTS" ]] && scp "$PTS" "${REMOTE}:${REMOTE_DIR}/workspace/bacnet/commissioning/points.csv" || true
+
+echo "Restarting bridge + commission containers..."
+ssh "$REMOTE" "cd ${REMOTE_DIR} && docker compose -f docker/compose.edge.yml restart bridge commission 2>/dev/null || docker compose restart bridge commission"
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)